pkg/interface/src/logic/state/settings.ts

```diff
@@ -60,7 +71,19 @@ const useSettingsState = createState<SettingsState>('Settings', {
   tutorial: {
     seen: true,
     joined: undefined
+  },
+  keyboard: {
```

Comment:
Suggestions:

History back: `ctrl + [`
History forward: `ctrl + ]`
Previous channel: `ctrl + ;`
Next channel: `ctrl + '`
Toggle sidebar: `ctrl + \`

Ergonomically:
![image](https://user-images.githubusercontent.com/748181/117219687-c7663100-add3-11eb-83cd-688c7e7ccd1a.png)

Reply:
I actually love these — @liam-fitzgerald whaddya think?

Reply:
Sounds good although I haven't used a normal keyboard in a minute so not sure how those would feel in practice

Reply:
What kind of keyboard ya using? 

I tested this on a thinkpad and macbook and a 70% mech keyboard, it's pretty cozy

Reply:
`~mirmus-dozryx/clack` is calling you @liam-fitzgerald

Reply:
Kinesis Advantage